docs/modules/publisher.md

```diff
@@ -64,14 +64,18 @@ Check out this [link](https://github.com/Azure/iotedge/issues/1745) for more inf
 
 ## Breaking changes in 2.6.x from 2.5.x and lower versions
 
-While we attempt to maintain backwards compatibility as much as possible.  This means you should use any publisher module as a drop in replacement for a previous version.   However - to support new functionality from 2.6.x forward we removed previously (up to version 2.5.x) available functionality, including
+We attempt to maintain backwards compatibility as much as possible.  This means you can use any publisher module as a drop-in replacement for a previous version. However, to support new functionality from 2.6.x forward we removed previously (up to version 2.5.x) available functionality:
```

Comment:
@barnstee, can you please delete double spaces in the document ?

Reply:
Fixed